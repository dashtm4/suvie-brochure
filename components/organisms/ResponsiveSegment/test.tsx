import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ResponsiveSegment } from '.'

describe('<ResponsiveSegment />', () => {
  it('should renders correct <ResponsiveSegment />', () => {
    const wrapper = shallow(<ResponsiveSegment />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
