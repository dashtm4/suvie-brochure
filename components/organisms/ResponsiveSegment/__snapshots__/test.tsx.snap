// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResponsiveSegment /> should renders correct <ResponsiveSegment /> 1`] = `
<Carousel
  afterSlide={[Function]}
  autoGenerateStyleTag={true}
  autoplay={false}
  autoplayInterval={3000}
  autoplayReverse={false}
  beforeSlide={[Function]}
  cellAlign="left"
  cellSpacing={0}
  className="pb-4"
  defaultControlsConfig={Object {}}
  disableAnimation={false}
  disableEdgeSwiping={false}
  dragging={true}
  easing="easeCircleOut"
  edgeEasing="easeElasticOut"
  enableKeyboardControls={false}
  frameOverflow="hidden"
  framePadding="0px"
  getControlsContainerStyles={[Function]}
  height="inherit"
  heightMode="max"
  keyCodeConfig={Object {}}
  onDragStart={[Function]}
  onResize={[Function]}
  pauseOnHover={true}
  renderAnnounceSlideMessage={[Function]}
  renderBottomCenterControls={[Function]}
  renderCenterLeftControls={null}
  renderCenterRightControls={null}
  scrollMode="remainder"
  slideIndex={0}
  slideListMargin={10}
  slideOffset={25}
  slideWidth={1}
  slidesToScroll={1}
  slidesToShow={1}
  speed={500}
  style={Object {}}
  swiping={true}
  transitionMode="scroll"
  vertical={false}
  width="100%"
  withoutControls={false}
  wrapAround={false}
/>
`;
